cluster-up/cluster/k8s-provider-common.sh

```diff
@@ -159,6 +159,13 @@ function wait_for_cdi_ready() {
     fi
 }
 
+# configure Prometheus to select kubevirt prometheusrules
+function configure_prometheus() {
```

Comment:
Does it take some time for the configuration to apply? 
If so please consider also implementing wait function.

Reply:
From what I tested it was a one second operation, and anyway it doesn't block any future operations, so not sure it's needed.

Reply:
I meant to whatever `spec.ruleSelector` and `spec.ruleNamespaceSelector` change will trigger.